### DISCUSSION TITLE
Load Existing Tables

### DISCUSSION THREAD

**Discussion by jordanparker6** [nocodb/nocodb#521] [Q&A] [answered]
I am trying to run Nocodb as a UI on an existing postgres db. After connecting it to the db, I cannot see the existing tables within a project. Does Nocodb offer this feature? I cannot see it in the docs.

Thanks

**Comment by o1lab** [comment]
Hey, that is supported. Could you please paste project info ?

- Recreate the project one more time to confirm the problem. This is since - if you would change the project name before saving it would have changed the schema name hence you might be looking at an empty schema.
- Is it the public schema in which tables are present ? if not then this is how you [connect to it](https://www.youtube.com/watch?v=rS0NRwRCjx0&t=1s).
- Please reach out [on discord](https://discord.gg/5RgZmkW) - Im happy to help.

**Reply by jordanparker6** [reply]
Thanks for your help. Unfortunately, I cannot connect still.

I am trying to connect to public schema. I have added "searchPath": ["public"] to the settings.json. Still no dice...

I have created a seperate user for nocodb in postgres. Would this be creating any issues?

**Reply by o1lab** [reply]
Reach out on our discord - happy to help : https://discord.gg/5RgZmkW

**Comment by o1lab** [comment, marked as answer]
This was an issue with connected database ```user``` not having the permission to read the tables.
